hintrc

```diff
@@ -0,0 +1,962 @@
+[33mcommit d954e7827b016cdff87960a9d0163c33d6a10fd9[m
```

Comment:
This is a local file that shouldn't be committed into the repository. If you remove this file this can be merged with master.